website/pages/en/index.js

```diff
@@ -184,17 +190,45 @@ class Index extends React.Component {
                 <div className="blockImage">
                   <div>
                     <MarkdownBlock>
-                      {lambdaExample}
+                      {XHPExample}
                     </MarkdownBlock>
                   </div>
                 </div>
                 <div className="blockContent">
                   <h2>
-                    Lambdas
+                    XHP
                   </h2>
                   <div>
                     <MarkdownBlock>
-                      Lambdas succinctly allow definition of first-class functions.
+                      **[XHP](https://docs.hhvm.com/hack/XHP/introduction)** provides a native XML-like representation of output (e.g., HTML) and allows UI code to be typechecked, automatically avoiding several common issues like cross-site scripting (XSS) and double-escaping.
+                    </MarkdownBlock>
+                    <MarkdownBlock>
+                      XHP is _safe by default_: All variables are automatically escaped in a context-appropriate way.
+                    </MarkdownBlock>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </Container>
+          <Container padding={["bottom"]}>
+            <div className="gridBlock">
+              <div className="blockElement">
+                <div className="blockContent">
+                  <h2>
+                    Collaboration, Research, and Usage
+                  </h2>
+                  <div>
+                    <MarkdownBlock>
+                      HHVM and the Hack language are in active development. We are moving fast, making changes daily and releasing often. If you notice a regression in the typechecker or the runtime, please **[open issues](https://github.com/facebook/hhvm/issues/new)** when you find them.
+                    </MarkdownBlock>
+                    <MarkdownBlock>
+                      Learn about ongoing research and collaborations:
+                    </MarkdownBlock>
+                    <MarkdownBlock>
+                      * **[Jump-Start](https://engineering.fb.com/2021/03/03/developer-tools/hhvm-jump-start/)**: Jump-Start improves the performance of virtual machines at scale and has successfully been implemented in the HipHop Virtual Machine (HHVM), which powers not only Facebook.com but also many other sites across the web.
+                    </MarkdownBlock>
+                    <MarkdownBlock>
+                      * **[Zoncolan](https://engineering.fb.com/2019/08/15/security/zoncolan/)**: Facebook’s web codebase currently contains more than 100 million lines of Hack code, and changes thousands of times per day. With Zoncolan and its static analysis capabilities, security engineers scale their work by automatically examining Hack code and proactively detecting potentially dangerous security or privacy issues.
```

Comment:
Zoncolan is really cool, but it's not available to OSS users. Could we describe this as "Recent research" or something, to focus that it's academic output in this section?